index.html

```diff
@@ -1,11 +1,38 @@
-<html>
-
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
 <head>
-	<meta charset="utf-8">
-	<meta name="viewport" content="width=device-width">
-	<title>repl.it</title>
-	<link href="main.css" rel="stylesheet" type="text/css" />
-	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
-	<script src="main.js">
-	</script>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Calculator</title>
+    <link rel="stylesheet" href="main.css">
+    <script src="main.js" charset="utf-8" defer></script>
 </head>
+<body>
+    <div class="calculator-grid">
+        <div class="output">
+            <div data-previous-operand class="previous-operand">
+            </div> 
+            <div data-current-operand class="current-operand">
+            </div>
+        </div>
+        <button data-all-clear class="span-two">AC</button>
+        <button data-delete>DEL</button>
+        <button data-operation>&#247;</button>
+        <button data-number>1</button>
```

Comment:
Good start in exploring data attributes. This use of data attributes could be accomplished with a class. The most effective use of data attributes is to actually store data in an HTML element, which we didn't really talk about. It involves the use of something called a template literal. So, the calculator button `3` would have a data attribute like `data-number-3`. https://css-tricks.com/a-complete-guide-to-data-attributes/

Reply:
Then you could retrieve the number from the data attribute, versus from the inner text.

Reply:
I realized I made a mistake here. Should be `data-number=3` or `data-number=6` or whatever. The point is that the data attributes are used to store data that the user doesn't see. In this case, the inner text of the buttons is sufficient. But, if you had some kind of additional attribute, like a temporary sum or something, you could use a data attribute to store it.